resources/views/common/_navigationHeader.antlers.html

```diff
@@ -1,38 +1,60 @@
-<nav class="hidden text-xl uppercase lg:flex">
```

Comment:
![Screenshot 2024-11-08 at 16 19 57](https://github.com/user-attachments/assets/32854915-e504-440c-8331-2f1a12c5de76)

by making these changes you have broken the header for mobile, I also asked for you to add a search bar into the offcanvas?